extensions/ql-vscode/src/view/model-editor/ModelTypeDropdown.tsx

```diff
@@ -40,6 +32,17 @@ export const ModelTypeDropdown = ({
   modelingStatus,
   onChange,
 }: Props): JSX.Element => {
+  const options: Array<{ value: ModeledMethodType; label: string }> = [
+    { value: "none", label: "Unmodeled" },
+    { value: "source", label: "Source" },
+    { value: "sink", label: "Sink" },
+    { value: "summary", label: "Flow summary" },
+    { value: "neutral", label: "Neutral" },
+  ];
+  if (language === QueryLanguage.Ruby) {
+    options.push({ value: "type", label: "Type" });
+  }
+
```

Comment:
Can you make this a `useMemo` to avoid changing the identity of the array on every render? This should make it a little bit faster to render.